sources/settings-extensions/motd/motd.toml

```diff
@@ -0,0 +1,13 @@
+[extension]
+supported-versions = [
+    "v1"
+]
```

Comment:
Does this file need to be installed in the spec?

Reply:
It does need to be installed, but I would like to do so in a followup after completing https://github.com/bottlerocket-os/twoliter/issues/111. As it is now, I'd probably need to move the config to the RPM, but I consider it to be source code of the extension and think that it belongs in the crate.